New RSTUF Guide (#341)
repository-service-tuf/repository-service-tuf: docs/source/guide/deployment/planning/services.rst

########
Services
########

The RSTUF API is a service that exposes a web service. By default, the
container image exposes port ``80`` or port ``443`` when
:ref:`using HTTP certificates <guide/repository-service-tuf-api/Docker_README:(Optional) `SECRETS_RSTUF_SSL_CERT\`>`.

> Depending on your RSTUF use case, you might need to expose these ports
> externally.

Reviewer: ```suggestion
Depending on the specific RSTUF use case, it may become necessary to expose these ports
externally.
```

> Depending on the specific RSTUF use case, it may become necessary to expose these ports

Reviewer: ```suggestion
Depending upon the specific RSTUF use case, it may become necessary to expose these ports
```

> The RSTUF Worker is not a service but a workload. No port exposed.

Reviewer: "No port exposed."  Do you mean "No port should be exposed." or "No port is exposed."  What is the meaning of this phrase?

Author: No port is exposed (by RSTUF Worker).

> The :ref:`guide/deployment/planning/deployment:Required Infrastructure Services`
> expose its ports depending on the deployment. Those services are required to
> communicate only with the
> :ref:`guide/general/introduction:RSTUF Components`.

Reviewer: ```suggestion
The :ref:`guide/deployment/planning/deployment:Required Infrastructure Services`
expose their ports depending upon the deployment. Those services are only required to
communicate with the
:ref:`guide/general/introduction:RSTUF Components`.
```
